docs/modules/ROOT/pages/installation/platform/aks.cdoc

```diff
@@ -0,0 +1,34 @@
+Apache Camel K is a lightweight integration platform, born on Kubernetes,
+with serverless superpowers.
+
+# Camel K deployment in AKS 
```

Comment:
It should be more reader-friendly to tell what AKS stands for like other guides.